src/ci/bin/testSingleWorkflowRunner.sh

```diff
@@ -68,3 +68,16 @@ jq '{
 
 cmp <(jq -cS . actual.json) <(jq -cS . expected.json)
 popd > /dev/null
+
+# Test 4: application should return non-zero exit code when parsing of command-line arguments fails
+set +e
+java \
+    -jar "${CROMWELL_BUILD_CROMWELL_JAR}" \
+    run ./centaur/src/main/resources/standardTestCases/hello/hello.wdl \
+    ./centaur/src/main/resources/standardTestCases/hello/hello.inputs
+retVal=$?
+if [ $retVal -eq 0 ]; then
+    echo "ERROR: application exited with exit code 0 when invalid command-line arguments were provided"
```

Comment:
is it possible to assert that the invocation failed for invalid arguments and not some other reason?

Reply:
Yes, we can use specific exit code in case of invalid arguments.

Reply:
Could we also `tee` the standard out to a file and evaluate the file contents for specific strings? We do this earlier in this file, lines 20-24